Makefile

```diff
@@ -0,0 +1,175 @@
+		| awk ' BEGIN {FS = "!!!"}; { \
+			printf "  $(_TARGET)%-$(OPTION_SIZE)s$(_NORMAL)%s%-$(OPTION_OFFSET)s[default:%s]\n", $$1, $$4, "\n", $$2 \
+		}'
+
+.PHONY: targets
+targets:	## Display available targets and descriptions
+	@$(ECHO) "$(_HEADER)=== Makefile Targets [$(APP_NAME)] ===$(_NORMAL)\n"
+	@grep -v -E '.*(\?|\:\=).*$$' Makefile | grep -E '\#\#.*$$' \
+		| $(_SED) -r -e 's/([\sA-Za-z0-9\_\-]+)\:[\sA-Za-z0-9\_\-\.]+\s*\#{2,3}\s*(.*)\s*$$/\1!!!\2/g' \
+		| awk ' BEGIN {FS = "(:|###)+.*?## "}; \
+			/\#\#\#/ 	{printf "$(_SECTION)%s$(_NORMAL)\n", $$1;} \
+			/:/   	 	{printf "  $(_TARGET)%-$(TARGET_SIZE)s$(_NORMAL) %s\n", $$1, $$2;} \
+		  '
+	@$(ECHO) ""
+
+### Versioning Targets ###
+
+# Bumpversion 'dry' config
+# if 'dry' is specified as target, any bumpversion call using 'BUMP_XARGS' will not apply changes
+BUMP_XARGS ?= --verbose --allow-dirty	## Extra arguments to pass down to version control utility
+BUMP_XARGS := $(call clean_opt,$(BUMP_XARGS))
+ifeq ($(filter dry, $(MAKECMDGOALS)), dry)
+  BUMP_XARGS := $(BUMP_XARGS) --dry-run
+endif
+BUMP_CFG  ?= .bumpversion.cfg				## Bump version configuration (default recommended)
+BUMP_CFG  := $(call clean_opt,$(BUMP_CFG))
+BUMP_TOOL := bump2version
+BUMP_PATH := $(CONDA_ENV_PATH)/bin/$(BUMP_TOOL)
+BUMP_CMD  := $(BUMP_TOOL) --config-file "$(BUMP_CFG)"
+
+.PHONY: dry
+dry: $(BUMP_CFG)	## Run 'bump' target without applying changes (dry-run) (call: make VERSION=<x.y.z> bump dry)
+	@-$(ECHO) > /dev/null
+
+# run bumpversion with replacement of the release time value within its config for auto-update on future revisions
+.PHONY: bump
+bump: bump-check bump-install  ## Bump version using <VERSION> specified as input (call: make VERSION=<x.y.z> bump)
+	@-$(MSG_I) "Updating package version..."
+	@[ "${VERSION}" ] || ( $(MSG_E) "Argument 'VERSION' is not specified to bump version"; exit 1 )
+	@$(SHELL) -c ' \
+		PRE_RELEASE_TIME=$$(head -n 1 RELEASE.txt | cut -d " " -f 2); \
+		$(CONDA_CMD) $(BUMP_CMD) $(BUMP_XARGS) --new-version "${VERSION}" patch; \
+		POST_RELEASE_TIME=$$(head -n 1 RELEASE.txt | cut -d " " -f 2); \
+		echo "Replace $${PRE_RELEASE_TIME} → $${POST_RELEASE_TIME}"; \
+		$(_SED) -i "s/$${PRE_RELEASE_TIME}/$${POST_RELEASE_TIME}/g" $(BUMP_CFG); \
+		git add $(BUMP_CFG); \
+		git commit --amend --no-edit \
+	'
+
+.PHONY: bump-install
+bump-install:   ## Installs bumpversion if not detected in the environment
+	@-$(SHELL) -c '$(CONDA_CMD) test -f "$(BUMP_PATH)" || pip install $(PIP_XARGS) $(BUMP_TOOL)'
+
+.PHONY: bump-check
+bump-check:		## Verifies that required bumpversion files are found
+	@[ -f "$(BUMP_CFG)" ] || ( \
+		$(MSG_E) "Missing required file [$(BUMP_CFG)]. Run [make init-bump] or update BUMP_CFG accordingly."; \
+		exit 1 \
+	);
+
+.PHONY: version
+version:	## Display project version
+	@-$(MSG_I) "$(APP_NAME) version: $(APP_VERSION)"
+
+### Execution Targets ###
+
+SCRIPT ?= birdhouse/pavics-compose.sh	## Script to run the stack
```

Comment:
Wow, pavics-compose start stop too?!  Is this something you had in-house with CRIM now just making it public?

Reply:
Just a quick shortcut. Again, we could add more variations with auto piping of the xargs from `make` to `pavics-compose.sh` to call virtually anything, but I didn't go too crazy on it since it's not the main concern of the PR.